content/docs/alerting/configuration.md

```diff
@@ -128,6 +128,15 @@ current node.
 
 ```yaml
 [ receiver: <string> ]
+# The labels by which incoming alerts are grouped together. For example,
+# multiple alerts coming in for cluster=A and alertname=LatencyHigh would
+# be batched into a single group.
+#
+# To aggregate by all possible labels use '...' as the sole label name. 
```

Comment:
We also use ... to mean "more labels" so it'd be good to clarify the syntax is literal here

Reply:
thanks for the quick feedback, i rephrased it a bit